apps/openmw/engine.cpp

```diff
@@ -140,6 +138,7 @@ void OMW::Engine::frame(float frametime)
             mEnvironment.getMechanicsManager()->update(frametime,
                 guiActive);
         }
+        mEnvironment.getWorld()->markCellAsUnchanged();
```

Comment:
Okay, but I don't see hasCellChanged used anywhere in these changes? If you scrapped the idea, the markCellAsUnchanged move should be undone too. That flag is used by scripts so we'd need to do a careful review nothing was broken.